axelrod/tests/strategies/test_gambler.py

```diff
@@ -14,6 +13,7 @@
 
 tables = load_pso_tables("pso_gambler.csv", directory="data")
 C, D = axl.Action.C, axl.Action.D
+random = axl.RandomGenerator()
```

Comment:
Same comment here about using `axl._module_random` (or not)?

Reply:
Same answer. I don't want there to be any "magic" in how the RNGs work to prevent any stealthy reproducibility issues from creeping in, so being explicit here instead of relying on the module random is better.